samples/client/petstore/kotlin-allOff-discriminator/bin/main/org/openapitools/client/apis/BirdApi.kt

```diff
@@ -0,0 +1,122 @@
+/**
```

Comment:
@jo1gi thanks for the PR. where do these new kotlin files come from?

Reply:
I have no idea. Tried rerunning the sample generation commands.